### ISSUE TITLE
Breaks Spawn Eggs

### ISSUE THREAD

**Issue by curtlog** [DragonEggBedrockBreaking/VanillaDisable#5]
Spawn eggs cause an error instead of spawning a mob on use.

<details>
<summary>Error</summary>

```
[13:08:44] [Server thread/FATAL]: Error executing task on Server
org.spongepowered.asm.mixin.injection.callback.CancellationException: The call method_5894 is not cancellable.
	at org.spongepowered.asm.mixin.injection.callback.CallbackInfo.cancel(CallbackInfo.java:101) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at net.minecraft.class_1299.handler$zkg000$cancelSpawningFromItemStack(class_1299.java:703) ~[client-intermediary.jar:?]
	at net.minecraft.class_1299.method_5894(class_1299.java) ~[client-intermediary.jar:?]
	at net.minecraft.class_1826.method_7884(class_1826.java:89) ~[client-intermediary.jar:?]
	at net.minecraft.class_1799.method_7981(class_1799.java:240) ~[client-intermediary.jar:?]
	at net.minecraft.class_3225.method_14262(class_3225.java:344) ~[client-intermediary.jar:?]
	at net.minecraft.class_3244.method_12046(class_3244.java:1057) ~[client-intermediary.jar:?]
	at net.minecraft.class_2885.method_12547(class_2885.java:30) ~[client-intermediary.jar:?]
	at net.minecraft.class_2885.method_11054(class_2885.java:8) ~[client-intermediary.jar:?]
	at net.minecraft.class_2600.method_11072(class_2600.java:21) ~[client-intermediary.jar:?]
	at net.minecraft.class_3738.run(class_3738.java:18) ~[client-intermediary.jar:?]
	at net.minecraft.class_1255.method_18859(class_1255.java:151) ~[client-intermediary.jar:?]
	at net.minecraft.class_4093.method_18859(class_4093.java:23) ~[client-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_24306(MinecraftServer.java:780) ~[client-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_18859(MinecraftServer.java:162) ~[client-intermediary.jar:?]
	at net.minecraft.class_1255.method_16075(class_1255.java:125) ~[client-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_20415(MinecraftServer.java:762) ~[client-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_16075(MinecraftServer.java:756) ~[client-intermediary.jar:?]
	at net.minecraft.class_1255.method_5383(class_1255.java:110) ~[client-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_16208(MinecraftServer.java:740) ~[client-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:688) ~[client-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:270) ~[client-intermediary.jar:?]
	at java.lang.Thread.run(Thread.java:833) [?:?]
```

</details>

**Comment by DragonEggBedrockBreaking** [comment]
I know the issue, I will fix it tomorrow. Until then, open`.minecraft/config/vanilla-disable-mixin.properties` and add the line `mixin.spawning=false`. This will disable all spawning game rules (I.e. animal breeding, spawn eggs, mob spanner’s, and whether mobs can spawn at all), and should fix the issue.

**Comment by curtlog** [comment]
Yep, that fixes it.

**Comment by DragonEggBedrockBreaking** [comment]
Fixed in 3ec48038e25ef265dea720c782db5a3ca9657b1a. This version has still not been released. If you know how to build dev versions yourself, feel free to do so, otherwise wait until the upcoming release (either v1.1.1 or v1.2.0, haven't decided yet).

**Comment by yesn0t** [comment]
hello, i just found this issue and the above workaround seems to also fix crashes with mob spawners when used with mobs other than those covered by this mod (for example, husks, or rabbits)  
thanks and looking forwards to it x)

**Comment by DragonEggBedrockBreaking** [comment]
Fixed in 1.2.0: https://modrinth.com/mod/vanilla-disable/version/Fgt7Y6gP
After downloading this version, you can remove the line from the file, which re-adds the gamerules I mentioned.
